src/tensorinfo.cpp

```diff
@@ -142,6 +142,17 @@ ov::element::Type TensorInfo::getPrecisionFromDataType(tensorflow::DataType data
     }
 }
 
+ov::element::Type TensorInfo::getOvPrecision() const {
```

Comment:
This may not create conflict when merged with tensor info but should be checked & possibly removed then. Just a remark